lumicks/pylake/channel.py

```diff
@@ -122,21 +131,22 @@ class Continuous:
     dt : int
         Delta between two timestamps. Constant for the entire data range.
     """
-    def __init__(self, data, start, dt):
+    def __init__(self, data, start, dt, calibration=None):
         self._src_data = data
         self._cached_data = None
         self.start = start
         self.stop = start + len(data) * dt
         self.dt = dt
+        self._calibration = calibration
```

Comment:
Have you considered storing the calibrations in `Slice` instead of in the individual data sources? I see some small differences in the implementations of the calibrations for continuous and time series sources, but I think it would be a huge advantage for maintenance to make the behavior uniform and pull it up into `Slice`. 

If the reason for the different behavior of calibrations is the case of the empty slice, I'd be in favor of making the empty slice behavior more uniform rather than diverging the calibration behavior.

Reply:
It was the latter, but makes sense. Moved calibration into slice now.